### ISSUE TITLE
Shall I downgrade docker-ce?

### ISSUE THREAD

**Issue by adRn-s** [moncho/dry#146]
```
# dry
INFO[0000] Dry could not start: Error retrieving container list: Error response from daemon: client version 1.41 is too new. Maximum supported API version is 1.39 

# docker -v
Docker version 18.09.6, build 481bc77156
```

I looked into different closed issues, so I already tried running as `DOCKER_API_VERSION=1.39 dry -m`, but that didn't help.

**Comment by moncho** [comment]
Hmm you are running a quite old version already, not sure if downgrading docker-ce is going to help. But I wonder why the `DOCKER_API_VERSION` env variable is not working, could you please try with an older version of dry to see if it helps? With version [v0.10-beta.1](https://github.com/moncho/dry/releases/tag/v0.10-beta.1) for example.

**Comment by adRn-s** [comment]
Thanks, that solved it. The old version is using APIVersion 1.39 without needing the env var. I will close the issue now. Let me know if you want me to run some test to debug this.

**Comment by moncho** [comment]
Thanks @adRn-s, I might ask you at some point to try again with a new version with the problem fixed.

**Comment by parijke** [comment]
solved by upgrading Docker version 20.10.11
